en/docs/developer-guides/ai-for-integration/build-an-http-service-with-wso2-copilot.md

```
@@ -50,33 +50,46 @@ In this tutorial, you’ll create an HTTP service to add key-value pairs to a Re
-## Step 5: Configure the Redis client
+## Step 6: Configure the Redis client
 
 1. In the `Design View`, click on the **Configure** button on the top-right side.
 2. Set `redisHost` value to `localhost`.
 3. Set `redisPort` value to `6379`.   
-4. Set `redisPassword` value to `""`.   
 
 > **Note:** No need to set the above values if the configurable variables are generated with default values.
 
-   <a href="{{base_path}}/assets/img/developer-guides/ai-for-integration/http-service-with-copilot/config.gif"><img src="{{base_path}}/assets/img/developer-guides/ai-for-integration/http-service-with-copilot/config.gif" alt="Configurations" width="70%"></a>
+      <a href="{{base_path}}/assets/img/developer-guides/ai-for-integration/http-service-with-copilot/configuration.gif"><img src="{{base_path}}/assets/img/developer-guides/ai-for-integration/http-service-with-copilot/configuration.gif" alt="Configurations" width="70%"></a>
```

markdownlint-cli2 (0.20.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)